src/datagrid/DataGridUtils.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2018 - 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import {ICON_HEIGHT} from "./constants";
+import {FilterPosition} from "./DataGridFilter";
+
+/**
+ * Get X axis position value for supplied object
+ * @param {sting} position - Position prop, Accepted values right, left, center
```

Comment:
If the accepted values are fixed, then introduce a enum and use that enum instead of string as type for this prop